### PR TITLE
Clarify import vs. include for script loading; closes #57

### LINKED ISSUE #57: horizon client getting started guide
Migrated from rethinkdb/horizon#433 by @hnordt 

In http://horizon.io/docs/getting-started/ the docs say:

> However, if you’re using [Webpack][] or a similar build setup, or requesting the .js library at load time isn’t desirable, just add the client library as an NPM dependency (npm install @horizon/client).
>
> In your application, you’ll need to include the Horizon client file, and specify the Horizon port number when initializing the connection.

```js
<script src="localhost:8181/horizon/horizon.js"></script>

// Specify the host property for initializing the Horizon connection
const horizon = Horizon({host: 'localhost:8181'});
```

`In your application, you’ll need to include the Horizon client file, and specify the Horizon port number when initializing the connection.` shouldn't be `In your application, you’ll need to import the Horizon client file, and specify the Horizon port number when initializing the connection.`?

Also the code shouldn't be:

```js
import Horizon from '@horizon/client';

// Specify the host property for initializing the Horizon connection
const horizon = Horizon({host: 'localhost:8181'});
```

?

### PR DESCRIPTION
**Pull request by chipotle**

### CODE REVIEW

**Comment by chipotle** [comment]
Pinging @deontologician and @danielmewes — I don't know if this PR is actually accurate, given further comments on #57...

**Comment by deontologician** [comment]
I am not sure we should merge this yet. I don't know exactly what the import issues are, and they probably aren't resolved enough yet to want to document it.

**Comment by danielmewes** [comment]
@deontologician Do we know how this should work by now?

**Comment by chipotle** [comment]
I'm going to ping @dalanmiller and @deontologician about this one again.

File: getting-started.md [hunk 1]
```diff
@@ -311,19 +311,28 @@ While Horizon serves static files from the `dist` folder by default, you can use
...
 
-We recommend the first option, as that will prevent any possibly mismatches between the client library version and the Horizon server. However, if you're using [Webpack][] or a similar build setup, or requesting the `.js` library at load time isn't desirable, just add the client library as an NPM dependency (`npm install @horizon/client`).
-
-In your application, you'll need to include the Horizon client file, and specify the Horizon port number when initializing the connection.
-
-[webpack]: https://webpack.github.io/
+We recommend the first option, as that will prevent any possibly mismatches between the client library version and the Horizon server. In your application, you'll need to include the Horizon client file, and specify the Horizon port number when initializing the connection.
 
 ```html
 <script src="localhost:8181/horizon/horizon.js"></script>
```
Comment (deontologician) [inline getting-started.md]: This should just be `/horizon/horizon.js`

**Comment by deontologician** [comment]
Other than the script tag, I think this looks fine

**Comment by deontologician** [comment]
Also, yeah it works in Node now

**Pull request merged**